frontend/src/components/Warning.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+export default function Warning(props: {className:string}) {
```

Comment:
I think this component returns an icon?
If so, please update the name to `WarningIcon`, or something alike